src/index.js

```diff
@@ -1,12 +1,14 @@
 import _ from 'lodash'; //eslint-disable-line
 import './style.css';
+import setState from './modules/getstates.js'; //eslint-disable-line
```

Comment:
- [ ] Linters are helpful in ensuring that we have clean code and use the best practices so kindly remove this disable and please try to fix the error instead.